doc/manual/cookbooks/phase_diagram/doc/phase_diagram.tex

```diff
@@ -0,0 +1,80 @@
+\subsubsection{Phase diagram}
+
+\label{sec:phase-diagram}
+For many models it is useful to see how the material parameters vary as a function of pressure and temperature. In this cookbook, a plot of material values (e.g., density or viscosity) as a function of temperature (x axis) and pressure (y axis) is referred to as a phase diagram. 
+
+In this phase diagram cookbook, phase diagrams are generated from model runs and visualized with VISIT, but Paraview could also be used to view the same output. % intro
+It is a simple way to check the implementation in a material model with visible outputs.% motivation
+The idea lies in initiating a linear temperature variation along the x-axis in a box geometry, so as to view the two axes, x and y,  as pressure and temperature, respectively. % technique
+Here, I visualize a diagram of the phase transitions implemented in the ViscoPlastic material model, as well as a lookup table in the Steinberg material model% examples
+
+\paragraph{The input file.}
+You can find the input file to run this cookbook example in \url{cookbooks/phase_diagram.prm}
+
+\par The model domain is  800 km  by 800 km box. % geometry
+Initial temperature increases from 273 K on the left side to 2273 K on the right side. % initial conditions
+Pressure, on the other hand, increases from the top of the box to the bottom with a constant gradient.
+This is assured by assigning a constant density with zero expansivity.
+It differs from a classic phase diagram only in the direction of pressure increase.
+
+\par Two compositions of pyrolite and harzburgite are included in the model. % composition
+With each run, only one composition is assigned to the whole domain in order to visualize the diagrams of these two separately.
+
+\par The material model is then set up to mimic mantle phase transitions at 410, 520, and 660 km.% setup of mantle transitions
+Details of phase transitions are taken from \cite{billen2018decoupling}.
+A trick of assigning the values of density to the field of thermal capacity is used here [MIB: How is this done? can you be more explicit]. % trick
+This serves the goal of visualizing values of reference densities of phases with the 'real density' assigned with a constant value in the model.
+Inputs for this material model is listed here:
+\lstinputlisting[language=prmfile]{cookbooks/phase_diagram/doc/material_model.prm.out}
+
+
+\paragraph{Results}
+
+\par Visualization of the model results yields a phase diagram of a pyrolitic mantle (Figure \ref{fig:phase_diagram_ph_density}). % phase diagram
+The field shown here has the reference densities of the pyrolite phases, though settings of phase transitions are over-simplified.
+One may notice that 3 transitions (i.e. one in the olivine system, two in the spinel system) are included for the 660 interfaces, % 3 transitons, higher T
+and they need to be modified at a higher temperature.
```

Comment:
Can you explain this a bit more? I don't really understand what you mean in this sentence.